### PR TITLE
IBX-1853: Used rebranded Ibexa name in codebase

### PR DESCRIPTION
**Pull request by Nattfarinn**
## JIRA: https://issues.ibexa.co/browse/IBX-1853

### CODE REVIEW

File: src/bundle/DependencyInjection/Configuration.php [hunk 1]
```diff
@@ -33,7 +33,7 @@ public function getConfigTreeBuilder(): TreeBuilder
                     ->addDefaultsIfNotSet()
                     ->children()
                         ->arrayNode('powered_by')
-                            ->info('Control if system should generate a powered by header to promote eZ Platform usage')
+                            ->info('Control if system should generate a powered by header to promote Ibexa usage')
```
Comment (konradoboza) [inline src/bundle/DependencyInjection/Configuration.php:36]:
```suggestion
                            ->info('Control if system should generate a powered by header to promote Ibexa DXP usage')
```

File: src/bundle/Resources/config/services.yaml [hunk 2]
```diff
@@ -9,15 +9,15 @@ parameters:
...
-        ttl: "https://www.ibexa.co/software-information/licenses-and-agreements/ez-trial-and-test-license-agreement-ez-ttl-v2.1"
+        ttl: "https://www.ibexa.co/software-information/licenses-and-agreements/ibexa-trial-and-test-license-agreement-ibexa-ttl-v2.1"
         service_life: "https://support.ibexa.co/Public/Service-Life"
         support_service: "https://www.ibexa.co/services/support-maintenance"
         training_service: "https://www.ibexa.co/services/training"
         consulting_service: "https://www.ibexa.co/services/consulting-services"
-        ee_product: "https://www.ibexa.co/products"
-        install_ee: "https://doc.ibexa.co/en/{ez.release}/getting_started/install_ez_enterprise/"
+        ee_product: "https://ibexa.co/products"
+        install_ee: "https://doc.ibexa.co/en/{ez.release}/getting_started/install_ibexa_dxp/"
```
Comment (konradoboza) [inline src/bundle/Resources/config/services.yaml:18]:
```suggestion
        install_ee: "https://doc.ibexa.co/en/{ibexa.release}/getting_started/install_ibexa_dxp/"
```

File: src/bundle/Resources/config/services.yaml [hunk 3]
```diff
@@ -9,15 +9,15 @@ parameters:
...
         doc: "https://doc.ibexa.co"
-        update: "https://doc.ibexa.co/en/latest/updating/updating_ez_platform/"
+        update: "https://doc.ibexa.co/en/latest/updating/updating_ibexa/"
```
Comment (konradoboza) [inline src/bundle/Resources/config/services.yaml:20]:
```suggestion
        update: "https://doc.ibexa.co/en/latest/updating/update_ibexa_dxp/"
```

File: src/bundle/SystemInfo/Collector/EzcHardwareSystemInfoCollector.php [hunk 4]
```diff
@@ -25,7 +25,7 @@ public function __construct(EzcSystemInfoWrapper $ezcSystemInfo)
     }
 
     /**
-     * Collects information about the hardware eZ Platform is installed on.
+     * Collects information about the hardware Ibexa is installed on.
```
Comment (konradoboza) [inline src/bundle/SystemInfo/Collector/EzcHardwareSystemInfoCollector.php:28]:
```suggestion
     * Collects information about the hardware Ibexa DXP is installed on.
```

File: src/bundle/SystemInfo/Collector/EzcPhpSystemInfoCollector.php [hunk 5]
```diff
@@ -25,7 +25,7 @@ public function __construct(EzcSystemInfoWrapper $ezcSystemInfo)
     }
 
     /**
-     * Collects information about the PHP installation eZ Platform is using.
+     * Collects information about the PHP installation Ibexa is using.
```
Comment (konradoboza) [inline src/bundle/SystemInfo/Collector/EzcPhpSystemInfoCollector.php:28]:
```suggestion
     * Collects information about the PHP installation Ibexa DXP is using.
```

File: tests/bundle/SystemInfo/Collector/_fixtures/corrupted_composer.json [hunk 6]
```diff
@@ -6,12 +6,12 @@
     "type": "project",
     "authors": [
         {
-            "name": "eZ dev-team & eZ Community",
+            "name": "eZ dev-team & Ibexa Community",
```
Comment (konradoboza) [inline tests/bundle/SystemInfo/Collector/_fixtures/corrupted_composer.json:9]:
```suggestion
            "name": "Ibexa Engineering & Ibexa Community",
```

**Review by konradoboza — CHANGES_REQUESTED** [review]

**Review by konradoboza — APPROVED** [review]

File: tests/bundle/SystemInfo/Collector/JsonComposerLockSystemInfoCollectorTest.php [hunk 7]
```diff
@@ -40,7 +40,7 @@ public function testCollectWithMinimumStability(): void
                     'license' => 'GPL-2.0',
                     'stability' => 'dev',
                     'dateTime' => new \DateTime('2016-02-28 14:30:53'),
-                    'homepage' => 'http://share.ez.no',
+                    'homepage' => 'http://share.ibexa.co',
```
Comment (alongosz) [inline tests/bundle/SystemInfo/Collector/JsonComposerLockSystemInfoCollectorTest.php:43]: This URL is incorrect
```suggestion
                    'homepage' => 'https://ibexa.co',
```

File: tests/bundle/SystemInfo/Collector/_fixtures/composer.lock [hunk 8]
```diff
@@ -155,7 +155,7 @@
                 "GPL-2.0"
             ],
             "description": "Kernel used by ezsystems/ezplatform and derivatives. Provides the Content Repository, its APIs, and the application's Symfony framework integration.",
-            "homepage": "http://share.ez.no",
+            "homepage": "http://share.ibexa.co",
```
Comment (alongosz) [inline tests/bundle/SystemInfo/Collector/_fixtures/composer.lock:158]:
```suggestion
            "homepage": "https://ibexa.co",
```

File: tests/bundle/SystemInfo/Collector/_fixtures/corrupted_composer.lock [hunk 9]
```diff
@@ -155,7 +155,7 @@
                 "GPL-2.0"
             ],
             "description": "Kernel used by ezsystems/ezplatform and derivatives. Provides the Content Repository, its APIs, and the application's Symfony framework integration.",
-            "homepage": "http://share.ez.no",
+            "homepage": "http://share.ibexa.co",
```
Comment (alongosz) [inline tests/bundle/SystemInfo/Collector/_fixtures/corrupted_composer.lock:158]:
```suggestion
            "homepage": "https://ibexa.co",
```

**Review by alongosz — CHANGES_REQUESTED** [review]

**Pull request merged**